source/Light.cpp

```diff
@@ -0,0 +1,139 @@
+#include "Light.hpp"
+#include "World.hpp"
+#include "Script.hpp"
+#include "Native.hpp"
+
+namespace GTA
+{
+	using namespace System;
+
+	Light::Light(Math::Vector3 position, Drawing::Color color, float range, float intensity)
+	{
+		_isEnabled = false;
```

Comment:
Shouldn't the light be initially enabled? When creating a new object, lot of users would expect it being initialized. (just my thoughts)  Maybe another constructor overload would be good